packages/vats/test/fake-bridge.test.js

```diff
@@ -0,0 +1,46 @@
+import { test } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
+
+import { makeIssuerKit } from '@agoric/ertp';
+import { withAmountUtils } from '@agoric/zoe/tools/test-utils.js';
+import { makeHeapZone } from '@agoric/zone';
+import { E } from '@endo/far';
+import { buildRootObject as buildBankVatRoot } from '../src/vat-bank.js';
+import { makeFakeBankBridge } from '../tools/fake-bridge.js';
+
+const issuerKit = makeIssuerKit('IST');
+const stable = withAmountUtils(issuerKit);
+const ten = stable.units(10);
+
+test('balances', async t => {
+  const zone = makeHeapZone();
+
+  const bankBridge = makeFakeBankBridge(zone, {
+    balances: { agoric1fakeBridgeAddress: { uist: ten.value } },
+  });
+
+  const bankManager = await buildBankVatRoot(
+    undefined,
+    undefined,
+    zone.mapStore('bankManager'),
+  ).makeBankManager(bankBridge);
+
+  await E(bankManager).addAsset('uist', 'IST', 'Inter Stable Token', issuerKit);
+
+  const bank = await E(bankManager).getBankForAddress(
+    'agoric1fakeBridgeAddress',
+  );
+
+  const istPurse = await E(bank).getPurse(issuerKit.brand);
+  const notifier = E(istPurse).getCurrentAmountNotifier();
+  const updateRecord = await E(notifier).getUpdateSince();
+
+  t.like(updateRecord.value, ten);
+  const deposited = await E(istPurse).deposit(
+    stable.mint.mintPayment(stable.units(2)),
```

Comment:
This is where things go south.  You're minting ERTP assets out of thin air, and that's an operation that vbank cannot track.  The `issuerKit.mint` must be managed exclusively by the vbank.

What you can do is withdraw an ERTP payment from, say, `istPurse` to trigger a `VBANK_GRAB` and a balance notification.  Then, you can deposit that payment back into `istPurse` to trigger a `VBANK_GIVE` and another balance notification.

For more realism, consider creating a second `bank` object and transferring payments between the two.